R/utility.R

```diff
@@ -267,36 +289,36 @@ createOutputFile <- function(filePattern, url){
       uploadCondition = "taskCompletion"
     )
   )
-  
+
```

Comment:
Lots of line ending changes. Is there a linter we can use to keep everyone consistent?